CONTRIBUTING.md

```diff
@@ -56,6 +56,8 @@ Specifically:
 When filing a pull request, you should be prepared to answer questions about your changes
 and to perform additional work on the changes in response to review feedback.
 
+.xlf files do not need to be modified.  The English xlf is generated by the TranslationApp and then uploaded to Transifex when a release is generated.  The other languages are generated from Transifex site.  
```

Comment:
Sidenote: This is using "American" double space after periods where most using single space.
On the other hand, I would propose to use US English consistently, not to mix.
(The documentation is mostly in British English now.)
Maybe should be added to a style guide.